pkg/util/http.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package util
+
+import (
+	"context"
+	"io"
+	"log"
+	"net/http"
+)
+
+func SendHTTPRequest(ctx context.Context, method, contentType, endpoint string, body io.Reader) ([]byte, error) {
```

Comment:
can we remove `log.Print` from this method (assuming these were added for debugging)